libproxy/cmake/pkgconfig.cmk

```diff
@@ -1,4 +1,4 @@
-if(NOT WIN32 AND NOT APPLE)
+if(NOT WIN32)
   find_package(PkgConfig)
```

Comment:
This will require anyone wanting to build libproxy on Apple system to have pkg-config available. Instead `include(cmake/pkgconfig.cmk)` should be added just before https://github.com/libproxy/libproxy/blob/09fce547dfadc824686aa0bbf4935a04491a0790/libproxy/cmake/modules/pacrunner_duktape.cmk#L10  (conditional on `WITH_DUKTAPE)`, and Duktape should be disabled by default on Darwin https://github.com/libproxy/libproxy/blob/09fce547dfadc824686aa0bbf4935a04491a0790/libproxy/cmake/modules/pacrunner_duktape.cmk#L7

Reply:
FWICT if pkg-config is not found it will just not do anything. So this should work as expected already.

Reply:
Duktape is disabled if not found, which is the case in nixpkgs.

Note that we also have #76 which aims to make Apple's JavaScriptCore optional when packaging webkitgtk.